datasets/mtbs/mtbs-example.ipynb

```diff
@@ -599,7 +599,7 @@
    ],
    "source": [
     "da = rioxarray.open_rasterio(\n",
-    "    pc.sign(item).assets[\"burn-severity\"].href, chunks={\"x\": 2560, \"y\": 2560}\n",
+    "    item.assets[\"burn-severity\"].href, chunks={\"x\": 2560, \"y\": 2560}\n",
```

Comment:
This is the change in mtbs. We were signing it twice, since `item` is already signed. https://github.com/microsoft/planetary-computer-sdk-for-python/issues/21